Civi/Eck/API/Entity.php

```diff
@@ -159,7 +159,7 @@ public static function getEckAfforms($event) {
         'is_dashlet' => FALSE,
         'is_public' => FALSE,
         'is_token' => FALSE,
-        'permission' => 'access CiviCRM',
+        'permission' => 'access CiviCRM', // TODO: New ECK permissions.
```

Comment:
This is how you do it:

```suggestion
        'permission' => ['view any eck entity', 'view custom entity ' . $entityType['name']],
        'permission_operator' => 'OR',
```